transmit-lib/src/cmr/transmit/echo/tokens.clj

```diff
@@ -48,9 +48,9 @@
                                              :form-params {:id token}})]
       (case (int status)
         200 (get-in parsed [:token_info :user_name])
-        401 (errors/throw-service-error
+        401 (errors/throw-service-errors
              :unauthorized
-             (format "Token %s does not exist" token))
+             (:errors (json/decode body true)))
```

Comment:
If someone tries to use a Launchpad token (or anything longer than 100 characters) in one of the environments will they get a 401 back? I just want to make sure we don't throw a 500 error because we're not actually set up to use Launchpad anywhere without the manual cacerts install.

Reply:
Sounds like we need an explicit test for this ...

Reply:
We will return 401 back with "Token [xxx] does not exist." in this case.